force-app/main/default/classes/Data Recipes/DynamicSOQLRecipes.cls

```diff
@@ -17,17 +17,14 @@ public with sharing class DynamicSOQLRecipes {
     /**
      * @description demonstrates a simple dynamic SOQL query where the query is
      * defined in Apex
-     * @return List<Account> with security enforced
+     * @return List<Account> with security enforced using USER_MODE
      * @example
      * System.debug(DynamicSOQLRecipes.simpleDynamicSOQLQuery());
      **/
     public static List<Account> simpleDynamicSOQLQuery() {
-        String queryString =
-            'SELECT Id ' +
-            'FROM Account ' +
-            'WITH SECURITY_ENFORCED LIMIT 1 ';
+        String queryString = 'SELECT Id ' + 'FROM Account LIMIT 1';
```

Comment:
We could merge this into a single string without the concatenation.
Also, no need for `escapeSingleQuotes` in this case (see next line).

Reply:
Good catch @pozil! Just fixed it.